app/electron/main.ts

```diff
@@ -571,10 +571,11 @@ function startElecron() {
     to the default browser
     */
     mainWindow.webContents.on('will-navigate', (event, url) => {
-      if (url.startsWith(startUrl)) {
-        return;
+      if (!url.startsWith(startUrl)) {
+        event.preventDefault();
+        shell.openExternal(url);
       }
-      shell.openExternal(url);
+      return;
```

Comment:
You inverted the logic here but the return with this logic is not doing anything.
A more understandable change would be to just add the preventDefault, which seems to be the missing bit here.

Reply:
got it, made the change